solidity/ts/tests/test.ts

```diff
@@ -536,6 +536,81 @@ describe('Contract Test Suite', () => {
 		assert.strictEqual(finalShareBalances[2], 16n, `User did not receive excess Yes shares`)
 	})
 
+
+	test('canCloseOut', async () => {
```

Comment:
We should probably have a test for each of the following scenarios:
1. Close out with profit.
2. Close out with loss.
3. Close out neutral (probably hardest test to write).
4. Close out without any invalid shares (should balance your yes/no).
5. Close out with only invalid shares (should revert or have no effect).
6. Close out with pre-existing shares of both YES and NO, at profit.
7. Close out with pre-existing shares of both YES and NO, at loss.